File: fibonnaci.py
def fib(n):
    if n<=1:
        return n
    else:
        return fib(n-1) + fib(n-2)

n=int(input("enter a number:"))
print("the fibonacci number of",n,"is",fib(n))




"""@author: github.com/isacarcanjo"""

File: fibonnaci.ts

let fibonacci = (n: number): number => {
  if (n == 0) {
    return 0;
  }
  if (n == 1) {
    return 1;
  }
  return fibonacci(n - 1) + fibonacci(n - 2);
};

console.log(fibonacci(6));




"""@author: github.com/isacarcanjo"""

